Day5/src/constants.js

```diff
@@ -0,0 +1,2 @@
+export const title = "Avengers";
```

Comment:
Usually people do like, c`onst CONSTANT_NAME = value` so that when someone reads your code it is clear that this is a variable just coming from somewhere like constants.js